Lib/test/test_exceptions.py

```diff
@@ -2052,11 +2057,32 @@ def test_assertion_error_location(self):
                     'AssertionError',
                 ],
             ),
-            ('assert 1 > 2, "message"',
+            ('assert 1 > 2, "messäge"',
```

Comment:
These tests are not relevant to compiler errors. I updated/added them just to test that tracebacks for other errors also extract non-ASCII lines correctly.